src/main/java/com/conveyal/gtfs/loader/Table.java

```diff
@@ -98,15 +102,19 @@ public Table (String name, Class<? extends Entity> entityClass, Requirement requ
     }
 
     public static final Table AGENCY = new Table("agency", Agency.class, REQUIRED,
-        new StringField("agency_id",  OPTIONAL), // FIXME? only required if there are more than one
-        new StringField("agency_name",  REQUIRED),
-        new URLField("agency_url",  REQUIRED),
-        new StringField("agency_timezone",  REQUIRED), // FIXME new field type for time zones?
+        new StringField("agency_id",  OPTIONAL).requireConditions(
+            // If there is more than one agency, the agency_id must be provided
+            // https://developers.google.com/transit/gtfs/reference#agencytxt
+            new ConditionalRequirement("agency_id", ROW_COUNT_GREATER_THAN_ONE)
+        ).foreign(),
```

Comment:
I really don't think we should be marking fields as foreign like this. There is already a mechanism in place that does this: `Field#isReferenceTo`. In the body of that method (which is called on a field like `route#agency_id`, we could just set the `Field#isForeign` to `true`.

Reply:
Didn't realize that this was helping out with weird cases like `zone_id`. My changes in #317 should account for this.